models_evaluation.py

```diff
@@ -0,0 +1,176 @@
+from os import listdir
+from os.path import isfile, join
+import sys
+import argparse
+import logging
+import pathlib
+
+import gensim
+import pandas as pd
+import itertools
+
+logger = logging.getLogger("fasttext")
+
+
+def questions_reading(analogies):
+    """
+    Reads all type of questions to dict.
+    :param analogies: string
+    :return: dict
+    """
+    analogy_dict = dict()
+    with open(analogies, "r") as analogy_f:
+        key = None
+        for line in analogy_f:
+            if line in ['\n', '\r\n']:
+                continue
+            if line.startswith(":"):
+                # new section starts
+                key = line.rstrip()
+                analogy_dict[key] = list()
+            else:
+                analogy_dict[key].append(line[:-1].split("\t"))
+    return analogy_dict
+
+
+def analogy_evaluation(models_path, analogies, first_n):
+    """
+    Compute performance of the model on an analogy test set.
+    :param models_path: string
+    :param analogies: dict
+    :param first_n: int
+    :return: DataFrame
+    """
+    result = pd.DataFrame(
+        columns=["Model", "Type of questions", "Accuracy1 (for present questions)", "Accuracy2 (for all questions)",
+                 "Number of questions", "Number of present questions", "No words in vocabulary"])
+
+    for file in listdir(models_path):
```

Comment:
Just as an idea. We might use `glob.glob` here, in that case it'll be easy to use it more granularly.

I.e, if you have a directory with 10 models, you can pass filemask like:
`/mydir/*cbow*.*` to test only cbow's, etc.

Or even point the script to one particular model (to be honest, I was pissed off by default behavior of the original script, when testing models one by one by creating separate dir for each file)